ArchiSteamFarm/Bot.cs

```diff
@@ -439,16 +439,33 @@ internal sealed class Bot {
 		}
 
 		internal static string ResponseStatus(string botName) {
+			Bot bot;
+			string result="";
 			if (string.IsNullOrEmpty(botName)) {
 				return null;
+			} else {
```

Comment:
```return``` above exits function already, ```else``` is pointless and only increases nesting, which results in decreasing code readability.